src/workflows-app/SubmissionDetails.test.js

```diff
@@ -199,22 +97,22 @@ describe('Submission Details page', () => {
 
     const headers = within(rows[0]).queryAllByRole('columnheader');
```

Comment:
Not necessary in this PR, but I'd also suggest changing these to `getAllByRole` so that tests fail faster when things aren't as expected.